Automattic/woocommerce-payments
# Revert "Add shortened customer bank statement"

#### Changes proposed in this Pull Request

We've stumbled upon questions on #3643 that may require changes on how we save the short statement descriptor in the settings. In order to avoid messing up with the settings structure, and generate compatibility issues, this PR is reverting the changes that added the shortened statement descriptor field.









#### Testing instructions

In case you'd like to double-check:

1. Go to the [WCPay settings](http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments).
2. There is no `Shortened customer bank statement` field below the `Full bank statement`.
3. Open the dev tools and reload the page.
4. In the `GET /wp-json/wc/v3/payments/settings` request, there is no `is_short_statement_descriptor_enabled` and `short_statement_descriptor` in the response.
5. Save the settings.
6. In the `POST /wp-json/wc/v3/payments/settings` request, there is no `is_short_statement_descriptor_enabled` and `short_statement_descriptor` in the payload.





-------------------


**Post merge**
